Add landing pages (#1801)
spryker/spryker-docs: docs/scos/dev/architecture/module-api/module-api.md

---
title: Module API
description: Learn about Module APIs, performance and scalability, semantic versioning, and Composer constrains for customer modules.
last_updated: Apr 25, 2023
template: concept-topic-template
---

> In this section, you can find the following documents:

Author: ```suggestion
In this section, you can learn about module APIs, performance and scalability, semantic versioning, and Composer constraints for customer modules. For specific information, see these documents:
```

> In this section, you can learn about module APIs, performance and scalability, semantic versioning, and Composer constraints for customer modules. For specific information, see these documents:

Reviewer: ```suggestion
This section is about module APIs, performance and scalability, semantic versioning, and Composer constraints for customer modules. For specific information, see these documents:
```
